angular2/test/angular2TextMask.spec.js

```diff
@@ -5,7 +5,7 @@ import packageJson from '../package.json'
 
 const MaskedInput = (isVerify()) ?
   require(`../${packageJson.main}`).default :
```

Comment:
Sorry about the confusion here. The failing test is due to this line not being renamed as well. The CI runs the tests on the compiled code in the `dist` folder and when that happens, this line is executed instead of the one below.